### PR TITLE
Updated placeholder profile picture

### PR DESCRIPTION
**Pull request by KateMengjunXia**
Previous image path was incorrect
https://nanopay.atlassian.net/projects/CPF/issues/CPF-171?filter=allopenissues&orderby=priority%20DESC&keyword=placeholder

### CODE REVIEW

File: src/foam/nanos/auth/User.js [hunk 1]
```diff
@@ -173,7 +173,7 @@ foam.CLASS({
       documentation: 'User\' profile picture.',
       view: {
         class: 'foam.nanos.auth.ProfilePictureView',
-        placeholderImage: 'images/ic-placeholder.png'
+        placeholderImage: 'images/merchant/ic-placeholder.png'
```
Comment (jlhughes) [inline src/foam/nanos/auth/User.js:176]: can we move the image please to images/ . 
merchant should be deprecated and everything moved.

**Review by jlhughes — CHANGES_REQUESTED** [review]

File: src/foam/nanos/auth/User.js (hunk 1 above)
Comment (TW80000) [inline src/foam/nanos/auth/User.js:176]: The merchant folder is part of our nanopay repo, not part of foam. However, an even bigger issue is that we have a direct reference to nanopay for the images service in foam at the moment: https://github.com/foam-framework/foam2/blob/master/src/services#L16

Comment (jlhughes) [reply src/foam/nanos/auth/User.js:176]: Good catch, this property/view can be a refinement on the nanopay side.

Comment (KateMengjunXia) [reply src/foam/nanos/auth/User.js:176]: I'll fix it. Thanks for pointing out.

Comment (KateMengjunXia) [reply src/foam/nanos/auth/User.js:176]: Moved this change to nanopay refinementm and copied the picture from images/merchant to images
https://github.com/nanoPayinc/NANOPAY/pull/6185

**Pull request closed without merging**